core/src/test/scala/doric/sem/TransformOpsSpec.scala

```diff
@@ -120,8 +120,10 @@ class TransformOpsSpec
           )
       }
       error.getMessage should startWith(
-        "Found duplicate column(s) in given column names: `a`, `b`"
```

Comment:
why? this one is more accurate

Reply:
But depending the version of spark they append a ; at the end, and is not exactly the same value